src/main/java/ui/listpanel/ListPanel.java

```diff
@@ -152,11 +181,39 @@ public final void refreshItems() {
 
     }
 
+    private int getClosedIssueSize() {
+        int count = 0;
+        for (GuiElement element: getElementsList()){
+            if (!element.getIssue().isOpen()){
+                count++;
+            }
+        }
+        return count;
+    }
+
     /**
      * This function updates the information in the panel footer.
      */
     private void updateFooter() {
-        totalLabel.setText(String.format("Total: %d ", issueCount));
+
+        if (issueCount == 0){
+            totalIssueText.setText(String.valueOf(0));
+            hideFooterPanelStatsDetails(true);
+        } else {
+            openIssueText.setText(String.valueOf(openIssueCount));
+            closedIssueText.setText(String.valueOf(closedIssueCount));
+            totalIssueText.setText(String.valueOf(issueCount));
+            hideFooterPanelStatsDetails(false);
```

Comment:
Should be `DRYer` to extract a method to update the texts of all 3 stats text nodes, as they should be updated together (either all or none). So I think it's not a good idea to just update the `totalIssueText` like in the `if` branch.